packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -265,8 +265,9 @@ export async function importBlock(
       // We want to track recent blocks coming from gossip, unknown block sync, and API.
       if (delaySec < SLOTS_PER_EPOCH * this.config.SECONDS_PER_SLOT) {
         this.metrics.importBlock.elapsedTimeTillBecomeHead.observe(delaySec);
-        if (delaySec > this.config.SECONDS_PER_SLOT / INTERVALS_PER_SLOT) {
-          this.metrics.importBlock.setHeadAfterFirstInterval.inc();
+        const cutOffSec = getSlotComponentDuration(this.config, this.config.ATTESTATION_DUE_BPS) / 1000;
```

Comment:
My preference would be always use `Math.ceil` or `Math.floor` whenever make a division on the `mili seconds` values.

Reply:
I think we will see `cutOffSec` being something like 3.999999 or 4.000001, so we would need to do `Math.round`. What do you think?